server/src/com/cloud/api/ApiResponseGsonHelper.java

```diff
@@ -71,4 +83,19 @@ public boolean shouldSkipField(FieldAttributes f) {
             return false;
         }
     }
+
+    private static class LogExclusionStrategy extends ApiResponseExclusionStrategy implements ExclusionStrategy {
+        public boolean shouldSkipClass(Class<?> arg0) {
+            return false;
+        }
+
+        public boolean shouldSkipField(FieldAttributes f) {
+            boolean skip = super.shouldSkipField(f);
```

Comment:
maybe peformance can be improved by handling thsi first and super afterwards (it is more code)